### DISCUSSION TITLE
How do I switch to light mode?

### DISCUSSION THREAD

**Discussion by grahamIT** [netbirdio/dashboard#359] [Q&A]
![image](https://github.com/netbirdio/dashboard/assets/68344654/acf335a7-37dd-457b-b521-2e01e0c9fa09)

I am trying to switch to light theme but that option seems to be greyed out. Does anyone know how to switch?

**Comment by braginini** [comment]
It is not yet implemented, that is why it is not available @grahamIT :)

**Comment by heisbrot** [comment]
Can be tracked on this PR: https://github.com/netbirdio/dashboard/pull/658
